qiskit_aer/backends/aerbackend.py

```diff
@@ -345,6 +392,65 @@ def _run(self, qobj, job_id='', format_result=True):
             return self._format_results(output)
         return output
 
+    def _run_direct(self, circuits, noise_model, config, job_id='', format_result=True):
+        """Run a job"""
+        # Start timer
+        start = time.time()
+
+        # Take metadata from headers of experiments to work around JSON serialization error
+        metadata_list = []
+        for idx, circ in enumerate(circuits):
+            if hasattr(circ, "metadata") and circ.metadata:
```

Comment:
Can this `hasattr` ever be false, `QuantumCircuit.metadata` should always exist, it just might be `None`